pkg/analyzer/analyzers/client.go

```diff
@@ -126,7 +126,7 @@ func (r AnalyzerRoundTripper) RoundTrip(req *http.Request) (*http.Response, erro
 
 // methodIsSafe is a helper method to check whether the HTTP method is safe according to MDN Web Docs.
 // https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods#safe_idempotent_and_cacheable_request_methods
-func methodIsSafe(method string) bool {
+func MethodIsSafe(method string) bool {
```

Comment:
nit: If we rename this to `IsMethodSafe`, it might read more clearly at the call site as a bool check. Otherwise, it could give the impression that we’re trying to convert the method into a safe one